commander/src/commands/transaction/sign.ts

```diff
@@ -56,13 +70,31 @@ export default class SignCommand extends BaseCommand {
 	`;
 
 	static examples = [
-		'transaction:sign \'{"amount":"100","recipientId":"13356260975429434553L","senderPublicKey":null,"timestamp":52871598,"type":0,"fee":"10000000", "nonce": "1", "recipientPublicKey":null,"asset":{}}\'',
+		'transaction:sign \'{"id":"17528738200145418850","type":8,"senderPublicKey":"c094ebee7ec0c50ebee32918655e089f6e1a604b83bcaa760293c61e0f18ab6f","nonce":"1","fee":"100000000","asset":{"data":"{"liskhq":"zug"}","amount":"100000000000","recipientId":"5553317242494141914L"}}\'',
+		'transaction:sign \'{"id":"17528738200145418850","type":8,"senderPublicKey":"c094ebee7ec0c50ebee32918655e089f6e1a604b83bcaa760293c61e0f18ab6f","nonce":"1","fee":"100000000","asset":{"data":"{"liskhq":"zug"}","amount":"100000000000","recipientId":"5553317242494141914L"}}\' --mandatory-key=215b667a32a5cd51a94c9c2046c11fffb08c65748febec099451e3b164452bca --optional-key=922fbfdd596fa78269bbcadc67ec2a1cc15fc929a19c462169568d7a3df1a1aa --number-of-signatures=2 --number-of-passphrases=2',
+		'transaction:sign \'{"id":"17528738200145418850","type":8,"senderPublicKey":"c094ebee7ec0c50ebee32918655e089f6e1a604b83bcaa760293c61e0f18ab6f","nonce":"1","fee":"100000000","asset":{"data":"{"liskhq":"zug"}","amount":"100000000000","recipientId":"5553317242494141914L"}}\' --mandatory-key=215b667a32a5cd51a94c9c2046c11fffb08c65748febec099451e3b164452bca --optional-key=922fbfdd596fa78269bbcadc67ec2a1cc15fc929a19c462169568d7a3df1a1aa --number-of-signatures=2 --passphrase="inherit moon normal relief spring bargain hobby join baby flash fog blood" --passphrase="wear protect skill sentence lift enter wild sting lottery power floor neglect"',
```

Comment:
I think example is too big that it's hard to read. maybe we can simplify them so that the important arguments and flags are clear